app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,35 @@
+.required-loan-types
+  label = t('loan_questions.required_loan_types')
```

Comment:
I must have been thinking of the JS styleguide which I think does specify
string styles. I'm fine with leaving it.

On Thu, Sep 1, 2016 at 9:00 PM, Adam Konner <notifications@github.com>
wrote:

> In app/views/admin/loan_questions/_loan_types.slim
> <https://github.com/sassafrastech/madeline_system/pull/127#discussion_r77279833>
> :
>
> > @@ -0,0 +1,35 @@
> > +.required-loan-types
> > +  label = t('loan_questions.required_loan_types')
>
> I've always used single quotes unless interpolation is needed. You've
> never said anything about it before. Our style guide references bbatsov's
> style guide, which doesn't specify (but uses single quote style itself):
> https://github.com/bbatsov/ruby-style-guide#strings
> Personally, I don't have a problem with inconsistency in quoting style. If
> you do, we can talk about it, but there's currently nothing in the dev
> procedures doc about it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/sassafrastech/madeline_system/pull/127/files/e14c62e5b9f01af59e256dc20c3b81de5303afa8#r77279833>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAq3cA9LsJeDFvjZz2SitVqWmRl5tI-3ks5ql3U5gaJpZM4JrPfo>
> .
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech

Resident, Touchstone Cohousing
touchstonecohousing.org